npm/private/exclude_package_contents_default.bzl

```diff
@@ -0,0 +1,49 @@
+"A default files exclude list for common packages."
```

Comment:
Lets document where this list came from, probably just a link to the yarn docs and/or code?